cg_lims/EPPs/udf/calculate/twist_get_volumes_from_buffer.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import sys
+from typing import List
+
+import click
+
+from cg_lims import options
+from cg_lims.get.artifacts import get_artifacts, get_latest_artifact
+
+LOG = logging.getLogger(__name__)
+
+
+@click.command("get-buffer")
```

Comment:
```suggestion
@click.command("twist-get-buffer")
```

We should decide on some kind of rule for naming EPP commands. I think we want workflow specific commands to indicate what workflow they belong to. We can try like this and maybe we will have to rethink about it in the future when we have more EPPs.